Makefile

```diff
@@ -50,11 +50,9 @@ src/devices/%/mod.rs: src/devices/%/mod.full.rs
 	@# Remove DEVICE_PERIPHERALS declaration and replace it with a reference
 	@# to the global version
 	@patch --no-backup-if-mismatch --quiet $@ patch/modrs.patch
-	@echo -e "\tGEN-VECTOR\t>macros/src/vector.rs"
```

Comment:
Please don't remove the `@echo` lines here and below.

Reply:
reverted